lib/pdk/config/analytics_schema.json

```diff
@@ -1,6 +1,6 @@
 {
   "definitions": {},
-  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$schema": "http://json-schema.org/draft/schema#",
```

Comment:
This URL returns 404, where as the original `http://json-schema.org/draft-06/schema#` downloads the schema.
Just checking this references the new file that you created `lib/pdk/config/task_schema.json` instead?

Reply:
This is a really good catch @jordanbreen28.

It looks to me that there was some confusion with what the schema url would actually be. A change was made to the json-schema gem that made the validator check for the url in this change... however it actually turned out that it was released under the original url.

[This pr](https://github.com/voxpupuli/json-schema/pull/388) is the fix. I'm going to help get that merged now and will be able to correct the url in this PR.